src/DataProtection/custom/Cmdlets/Platform/Restore/New-AzDataProtectionRestoreConfigurationClientObject.ps1

```diff
@@ -40,17 +40,21 @@ function New-AzDataProtectionRestoreConfigurationClientObject{
         ${ConflictPolicy},
 
         [Parameter(Mandatory=$false, HelpMessage='Namespaces mapping from source namespaces to target namespaces to resolve namespace naming conflicts in the target cluster.')]
-        [Microsoft.Azure.PowerShell.Cmdlets.DataProtection.Models.Api202301.KubernetesClusterRestoreCriteriaNamespaceMappings]
+        [Microsoft.Azure.PowerShell.Cmdlets.DataProtection.Models.Api20230501.KubernetesClusterRestoreCriteriaNamespaceMappings]
         ${NamespaceMapping},
 
         [Parameter(Mandatory=$false, HelpMessage='Restore mode for persistent volumes. Allowed values are RestoreWithVolumeData, RestoreWithoutVolumeData. Default value is RestoreWithVolumeData')]
         [System.String]
         [ValidateSet('RestoreWithVolumeData','RestoreWithoutVolumeData')]
-        ${PersistentVolumeRestoreMode}
+        ${PersistentVolumeRestoreMode},
+        
+        [Parameter(Mandatory=$false, HelpMessage='Hook reference to be executed during restore.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.DataProtection.Models.Api20230501.NamespacedNameResource[]]
```

Comment:
Please confirm whether a helper comdlet is needed for RestoreHookReference NamespacedNameResource[]

Reply:
Hi @BethanyZhou, it's a list of an object with two fields. we wouldn't need a separate command for now. However, an example would help, let me add an example.